CMakeLists.txt

```diff
@@ -11,6 +11,10 @@ set(CMAKE_C_STANDARD_REQUIRED ON)
 set(LPAC_CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 list(APPEND CMAKE_MODULE_PATH ${LPAC_CMAKE_MODULE_PATH})
 
+if(CMAKE_EXPORT_COMPILE_COMMANDS STREQUAL "")
+  set(CMAKE_EXPORT_COMPILE_COMMANDS ON CACHE BOOL "project default" FORCE)
+endif()
```

Comment:
Use `set(CMAKE_EXPORT_COMPILE_COMMANDS ON)` directly. Force overriding the cached variable is not a good idea. The developers should rebuild the build cache by themselves.

Reply:
OK. Fixed.